app/views/application/_flash.html.erb

```diff
@@ -1,13 +1,9 @@
-<% if flash[:error] %>
-  <div class="alert alert--error">
-    <%= flash[:error] %>
-  </div>
-<% elsif flash[:warning] %>
-  <div class="alert alert--warning">
-    <%= flash[:warning] %>
-  </div>
-<% elsif flash[:notice] %>
-  <div class="alert">
-    <%= flash[:notice] %>
-  </div>
-<% end %>
+<div class="alerts">
+  <% flash.each do |type, message| %>
+    <% next if type == "alert" %>
+    <div class="alerts__alert<%= " alerts__alert--#{type}" if ["warning", "error"].include? type.to_s %>">
```

Comment:
Can I propose to add the space in front of the `<%=` instead of in the string just for the sake of readability?

Reply:
Sure, the only reason I did it this way is because the class name ends up being `"alerts__alert "` by default.